api/app/wildfire_one/wfwx_post_api.py

```diff
@@ -11,6 +11,14 @@
 logger = logging.getLogger(__name__)
 
 WF1_FORECAST_POST_URL = f"{config.get('WFWX_BASE_URL')}/v1/dailies/daily-bulk"
+WF1_HTTP_ERROR = HTTPException(
+    status_code=status.HTTP_400_BAD_REQUEST,
```

Comment:
nit: The 400 series of codes are typically used when there is a problem with the client's request. It might be better to return a 500 or 503 indicating a server side issue.